CHANGELOG.md

```diff
@@ -27,4 +30,4 @@
 - add DatadogProbabilitySampler
 - add DatadogPropagator
 - add setup and usage instructions
-- add unit tests
\ No newline at end of file
+- add unit tests
```

Comment:
What is changing here?

Reply:
I must've removed the end line. My formatter is acting up